app/views/defence_requests/new.html.erb

```diff
@@ -82,6 +82,13 @@
 
 <script>
  $(document).ready(function () {
+
+  $(document).on('keydown', function(event) {
+    if($('.solicitor_results_list *').length > 0 && event.keyCode === 27){
```

Comment:
Not sure whether it'll be a problem but sometime different keyboards report different values: http://unixpapa.com/js/key.html

I think `ESC` is consistent though so it should be okay